en/python_introduction/README.md

```diff
@@ -252,7 +252,11 @@ This shows that you just created an empty dictionary. Hurray!
 
 Now, try writing the following command (try replacing your own information too):
 
-    >>> participant = {'name': 'Ola', 'country': 'Poland', 'favorite_numbers': [7, 42, 92]}
```

Comment:
I believe we are intoruducing whitespace to them, but there is no need to make anything harder at this point. But I won't be arguing too much. Can someone taka a voice in this case please so we have some more inoput? :)